server/settings/main.py

```diff
@@ -1,7 +1,7 @@
 """Main settings for USD on AYON server."""
-from ayon_server.settings import BaseSettingsModel, MultiplatformPathListModel
-from pydantic import Field
 
+from ayon_server.settings import BaseSettingsModel, MultiplatformPathListModel
+from pydantic import Field, validator
 
```

Comment:
Requires to replace all `Field` in the code with `SettingsField`.
```suggestion
from ayon_server.settings import (
    BaseSettingsModel,
    MultiplatformPathListModel,
    SettingsField,
)
from pydantic import validator

```

Reply:
fixed